lib/publiccloud/noprovider.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright 2018 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Helper class for dummy provider
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+package publiccloud::noprovider;
+use Mojo::Base 'publiccloud::provider';
+use testapi;
+use publiccloud::existing;
+
+has ssh_key => undef;
+has ssh_key_file => undef;
+
+sub init {
+    my ($self) = @_;
+    $self->SUPER::init();
+    $self->provider_client(publiccloud::existing->new());
+    $self->provider_client->init();
```

Comment:
```suggestion
```

Reply:
there is suggestion to remove this line , but line is still here , but you marked it as resolved . hm can you please clarify ?

Reply:
I'm sorry, I did not notice that. The line is gone now.